renovate.json

```diff
@@ -1,12 +1,19 @@
 {
-  "extends": [
-    "config:base",
-    "helpers:pinGitHubActionDigests"
-  ],
-  "packageRules": [
-    {
-      "packageNames": ["org.mockito:mockito-core"],
-      "schedule": ["on monday"]
-    }
-  ]
+   "extends":[
+      "config:base",
+      "helpers:pinGitHubActionDigests"
+   ],
+   "packageRules":[
+      {
+         "matchUpdateTypes":[
+            "minor",
+            "patch",
+            "pin",
+            "digest",
+            "fix"
```

Comment:
`fix` is not an allowed value based on the documentation over [here](https://docs.renovatebot.com/configuration-options/#matchupdatetypes).